.circleci/config.yml

```diff
@@ -1,41 +1,93 @@
 version: 2
+
+aliases:
+  restore_cache: &restore_cache
+    restore_cache:
+      name: Restore node_modules cache
+      keys:
+        - npm-cache-{{ checksum "package-lock.json" }}
```

Comment:
If you add `-npm-cache-` this can boost results even if package-lock.json is the same.